Packs/MapValuesTransformer/Scripts/MapValuesTransformer/MapValuesTransformer.py

```diff
@@ -0,0 +1,58 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import json
+
+
+def mapValues(value, input_values, mapped_values):
+    # Convert all to string
+    value = str(value) if type(value) not in [dict, list] else value
+    input_values[:] = [str(x) for x in input_values]
+    mapped_values[:] = [str(x) for x in mapped_values]
+
+    # If the provided input_value and mapper_values are not equal in length
+    # then return an error
+
+    if len(input_values) != len(mapped_values):
+        return_error('Length of input_values and mapped_values are not the same')
+
+    # Create a dictionary to look up values against
+    mapper = dict()
+    for index in range(0, len(input_values)):
+        mapper[input_values[index]] = mapped_values[index]
+
+    # If the input is a dictionary then attempt to convert each
+    # "key: value" as a string into the value in the mapper
+    if type(value) == dict:
+        new_dict = dict()
+        for k, v in value.items():
+            key_value = f"{k}: {v}"
```

Comment:
specific format can make some troubles.
Some users can pass key value without space.
So either check if mapper contains `f"{k}: {v}"` and `f"{k}:{v}"` or use regex for that

Reply:
If an invalid JSON format is input, it will be treated as a string. The input expects a valid type of either str or dict, otherwise it will treat (converT) the input as a string.